lib/rules/require-camelcase-or-uppercase-identifiers.js

```diff
@@ -86,7 +86,7 @@ module.exports.prototype = {
                     return;
                 }
 
-                if (prevToken && prevToken.value === '.') {
+                if (prevToken && (prevToken.value === '.' || prevToken.type === 'Identifier')) {
```

Comment:
Should we explicitly check for `get` and `set` as the values of the identifiers?